Added Project Dataset Descriptions (#18)
d3b-center/OpenPedCan-methods: content/06.methods.md

#### Data and code availability

> OpenPedCan harmonized, aggregated, and analysed data from multiple sources. 

Reviewer: ```suggestion
Within OpenPedCan (OPC), we harmonized, aggregated, and analyzed data from multiple sources. 
```

> First for pediatric cancer data, OpenPedCan used the data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers [@dbgap:phs000218.v23.p8]. 

Reviewer: ```suggestion
We harmonized data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET cohort) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers [@dbgap:phs000218.v23.p8]. 
```
really, all except TCGA/GTEX are ped can, so just suggesting we remove that bit here

> The second dataset is the cancer samples from the Gabriella Miller Kids First Pediatric Research Program (Kids First), a large-scale effort to accelerate research and gene discovery in pediatric cancers and structural birth defects [@dbgap:phs001436.v1.p1].

Reviewer: ```suggestion
We included already harmonized neuroblastoma samples from the Gabriella Miller Kids First (GMKF cohort) Pediatric Research Program, a large-scale effort to accelerate research and gene discovery in pediatric cancers and structural birth defects [@dbgap:phs001436.v1.p1].
```

> Additionaly, OPC includes the Open Pediatric Brain Tumor Atlas (OpenPBTA) Project data, an open science initiative led by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab and the Center for Data-Driven Discovery (D3B) at the Children’s Hospital of Philadelphia (CHOP), which genomically characterized pediatric brain tumor data from the Children’s Brain Tumor Network (CBTN), and the Pacific Pediatric Neuro-oncology Consortium (PNOC) [@dbgap:phs002517.v1.p1], [@doi:10.1016/j.xgen.2023.100340].

Reviewer: ```suggestion
Additionally, we re-harmonized all samples from the Open Pediatric Brain Tumor Atlas (OpenPBTA, PBTA cohort), an open science initiative led by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab and the Center for Data-Driven Discovery (D3B) at the Children’s Hospital of Philadelphia (CHOP), which genomically characterized pediatric brain tumor data from the Children’s Brain Tumor Network (CBTN), and the Pacific Pediatric Neuro-oncology Consortium (PNOC) [@dbgap:phs002517.v1.p1], [@doi:10.1016/j.xgen.2023.100340].
```

Reviewer: After this, we should also add additional data which is in the PBTA cohort: PBTA X01 (Resnick: https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick, CA267587-01), Chordoma Foundation data: https://www.chordomafoundation.org/, and mioncoseq data (UMich clinical data donated to CBTN).

> The last pediatric cancer dataset is Children's Hospital of Philadelphia (CHOP) P30 Panel data generated by CHOP's Division of Genomic Diagnostics (DGD) which includes tumor SNV and fusion panel data [@dbgap:phs002517.v1.p1].

Reviewer: ```suggestion
Finally, OPC includes the Children's Hospital of Philadelphia (CHOP) P30 Panel data generated by CHOP's Division of Genomic Diagnostics (DGD cohort) which includes fusion panel data [@dbgap:phs002517.v1.p1].
```

> In addition to pediatric cancer data, OpenPedCan pulled in adult data from large science consortiums as references.

Reviewer: ```suggestion
In addition to pediatric cancer data, OpenPedCan contains adult data from large science consortiums as references.
```

> For normal gene expression, GTEx [@dbgap:phs000424.v9.p2] was used, and for comparison with adult cancers, The Cancer Genome Atlas (TCGA) [@https://www.cancer.gov/tcga] was also included.

Reviewer: ```suggestion
For normal gene expression, GTEx [@dbgap:phs000424.v9.p2] was used, and for comparison with adult cancers, The Cancer Genome Atlas (TCGA) [@https://www.cancer.gov/tcga] was included.
```

#### Data and code availability

Within OpenPedCan (OPC), we harmonized, aggregated, and analyzed data from multiple sources. 

> We harmonized data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET cohort) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers [@dbgap:phs000218.v23.p8]. 

Reviewer: ```suggestion
We harmonized data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET cohort) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers ([dbGAP phs000218.v23.p8](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000218.v23.p8)). 
```
I don't think dbgap is coded as a short url - so we can do something like this where we link out

Reviewer: can you update the other links and we can merge and see if it works? 🤞

Author: There is a dbgap prefix in bioregistry, https://bioregistry.io/registry/dbgap, which is why I used that. But if that won't work, sure I'll update the links quick

Reviewer: oh awesome, I was looking at the USAGE markdown - let's try your way!

We included already harmonized neuroblastoma samples from the Gabriella Miller Kids First (GMKF cohort) Pediatric Research Program, a large-scale effort to accelerate research and gene discovery in pediatric cancers and structural birth defects [@dbgap:phs001436.v1.p1].
Additionally, we re-harmonized all samples from the Open Pediatric Brain Tumor Atlas (OpenPBTA, PBTA cohort), an open science initiative led by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab and the Center for Data-Driven Discovery (D3B) at the Children’s Hospital of Philadelphia (CHOP), which genomically characterized pediatric brain tumor data from the Children’s Brain Tumor Network (CBTN), and the Pacific Pediatric Neuro-oncology Consortium (PNOC) [@dbgap:phs002517.v1.p1; @doi:10.1016/j.xgen.2023.100340].

> Building on the work of OpenPBTA, OPC added the PBTA X01 data [@https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick], the Chordoma Foundation data [@https://www.chordomafoundation.org/], and the MI-ONCOSEQ Study [@https://www.pathology.med.umich.edu/mctp/mi-oncoseq-study], donated to CBTN by the University of Michigan, to the PBTA cohort.

Author: > After this, we should also add additional data which is in the PBTA cohort: PBTA X01 (Resnick: https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick, CA267587-01), Chordoma Foundation data: https://www.chordomafoundation.org/, and mioncoseq data (UMich clinical data donated to CBTN).

@jharenza does this work to add in the references above?
